src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/EventTypeAT.java

```diff
@@ -0,0 +1,170 @@
+package de.zalando.aruha.nakadi.webservice;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.config.NakadiConfig;
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.EventTypeSchema;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+
+public class EventTypeAT extends BaseAT {
```

Comment:
I think the number of tests here should be reduced to a minimum. Move most of the tests to unit testing with mocking.